brb_membership/LICENSE-BSD

```diff
@@ -0,0 +1,15 @@
+<<<<<<< HEAD
+Copyright 2019 MaidSafe.net limited.
+=======
+Copyright 2020 MaidSafe.net limited.
+>>>>>>> 1b80124... import dynamic membership code from brb crate (and before that bft-crdts)
```

Comment:
Conflicted committed here

Reply:
thx, I've fixed this in a separate PR in the brb_membership crate.